catalog/transformer/catalog-transformer-service-atom/pom.xml

```diff
@@ -112,7 +112,9 @@
                             com.ctc.wstx.stax, <!-- needed by FOMWriter -->
                             *
                         </Import-Package>
-                        <Export-Package/>
+                        <Export-Package>
+                            ddf.catalog.transformer.response.query.atom
```

Comment:
❓ Shouldn't this be removed from the `Private-Package` declaration if it's being exported?

And _why_ is it being exported?

Reply:
It's being exported so `AtomTransformer` can be instantiated in a downstream project with different properties

Reply:
Removed it from `Private-Package`

Reply:
So it's being constructed on the fly and not retrieved as a service implementation?

Or is it being retrieved and then having one of its four service properties updated on the fly? If the latter, wouldn't it be easy enough to replace this bundle with one that updates the blueprint file, and exposes this service implementation?

Reply:
It's being constructed as a new bean in a downstream project and then published as an additional service to the system there.